lib/helpers.coffee

```diff
@@ -33,3 +33,53 @@ module.exports =
         return syntax[0]
 
     return existingSyntax
+
+  ###*
+   * Attempts to resolve the root directory/project directory of the currently open file/editor instance
+   * @param {string} filePath - The currently active editor file path
+   * @return {Object|null} The project directory instance or null if no project root is found
+   ###
+  getRootDir: (filePath) ->
+    rootDir = null
+    atom.project.rootDirectories.forEach((dir) ->
```

Comment:
I was having issues when multiple projects were open in one window though. I'll double check again though.